app.js

```diff
@@ -6,6 +6,22 @@ const fs = require('fs');
 const uuid = require('uuid');
 const os = require('os');
 
+// For starter kit env.
+require('dotenv').config({
+  silent: true
+});
+let pEnv = process.env;
```

Comment:
we can use `const` here as we are not updating the value of `pEnv` afterward.

Reply:
yep, pushed a commit